jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/SamInitRunner.kt

```diff
@@ -85,13 +71,13 @@ object SamInitRunner {
             LOG.info { "SAM output: ${process.stdout}" }
         }
 
-        val subFolders = tempDir.listFiles()
+        val subFolders = tempDir.listFiles()?.toList() ?: emptyList()
 
-        assert(subFolders != null && subFolders.size == 1 && subFolders[0].isDirectory) {
+        assert(subFolders.size == 1 && subFolders[0].isDirectory) {
```

Comment:
could also make this [0] a first to match the change below

Reply:
Fixed